### ISSUE TITLE
Where is your codes come from?

### ISSUE THREAD

**Issue by rxlgq** [zhiyou720/SPL-experiment#1]
Hi, I am very interesting in the SPL, can you provide the relative papers in your codes and some illustration? Thanks. In the TRAINTER.py, why use the crossentropy_losses, and not the losses which your definition?

**Comment by zhiyou720** [comment]
> Hi, I am very interesting in the SPL, can you provide the relative papers in your codes and some illustration? Thanks. In the TRAINTER.py, why use the crossentropy_losses, and not the losses which your definition?


Hi, this repo is my school coursework. SPL(self paced learning) can be seen as a specific loss function which my classmate and I did some reasearch on it. At the begining, we chose github to do team work coding. But at last, we tried lots of different experiments which we did not use github. This code is a prototype which we modified it from the cifar-10 image recognition test, and I can remember details because of it took too long. Sorry.
